File: ht10-spring-mvc-spa/ht10-server/src/main/java/ru/otus/kulygin/dto/list/BookListDto.java
package ru.otus.kulygin.dto.list;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import ru.otus.kulygin.dto.BookDto;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class BookListDto {

    private List<BookDto> books;

}

File: ht12-spring-security/ht12-client/src/main/web/src/app/models/list/book-list-dto.ts
import {BookDto} from "../book-dto";

export class BookListDto {

    private _books: BookDto[] = [];

    constructor(books: BookDto[]) {
        this._books = books;
    }

    get books(): BookDto[] {
        return this._books;
    }

    set books(value: BookDto[]) {
        this._books = value;
    }

}